app/models/user.rb

```diff
@@ -238,10 +238,6 @@ def spam_flag(post, dry_run = false)
     flag :spam, post, dry_run
   end
 
-  def abusive_flag(post, dry_run = false)
-    flag :abusive, post, dry_run
-  end
-
```

Comment:
This method is never used.

Reply:
No harm in keeping it, though

Reply:
Are we sure that this works after the token-store migration?